autoria.py

```diff
@@ -0,0 +1,242 @@
+                   e.g. ''Легковые''
+        mark - (required) string with mark, like ''Renault''
+        model - (requred) sting with model, like ''Scenic''
+        bodystyle - (optional) string with bodystyle, e.g. ''Седан''
+        year_start_end - (optional) list with start and end mfr year,
+                        e.g. ''[2005, 2006]'', also one of years could be
+                        ''None'', e.g. ''[2005, None]''
+            <== note: make this parameter behave like on auto.ria: if there's
+            only start year end=current_year, vice versa - start=1950==>
+        state - (optional) string with state (city), e.g. ''Харьков''
+        ******************************************************************
+        The following search parameters are not in use right now.
+        Are to be added.
+        gear_type - (optional) list with gearshift types,
+                    e.g. ''[Механическая, Автоматическая]''
+        fuel - (optional) list with gasoline types,
+                e.g. ''[Бензин, Газ/Бензин]''
+        drive - (optional) list with drive types,
+                e.g. ''[Передний, Полный]''
+        'engine': 'engineVolume',
+        'options': 'options',
+        'mileage': 'raceInt',
+        'doors': 'door',
+        'state': 'state_id',
+        'city': 'city_id',
+        'carrying': 'carrying',
+        'seats': 'seats',
+        'color': 'color_id',
+        'custom': 'custom',
+        'damage': 'damage',
+        'credit': 'under_credit',
+        'confiscated': 'confiscated_car',
+        'on_repairment': 'onRepairParts',
+        ******************************************************************
+
+        Returns:
+            JSON-formatted output with data as described here:
+            https://goo.gl/WVCVi8
+    """
+    params_dict = {}
+
+    # Getting list of categories and selecting needed id
+    categories_list = get_categories()
+    selected_category = select_item(category, categories_list)
+    params_dict['main_category'] = selected_category
+
+    # Getting list of bodysyles and selecting needed id
+    bodystyles_list = get_bodystyles(selected_category)
+    selected_bodystyle = select_item(bodystyle, bodystyles_list)
+    params_dict['body_id'] = selected_bodystyle
+
+    # Getting list of marks and selecting needed id
+    marks_list = get_marks(selected_category)
+    selected_mark = select_item(mark, marks_list)
+    params_dict['marka_id'] = selected_mark
+
+    # Getting list of models and selecting needed id
+    models_list = get_models(selected_category, selected_mark)
+    selected_model = select_item(model, models_list)
+    params_dict['model_id'] = selected_model
+
+    # Getting list of states and selecting needed id
+    states_list = get_states()
+    selected_state = select_item(state, states_list)
+    params_dict['state_id'] = selected_state
+
+    # Creting a list from start and end years
+    params_dict['yers'] = years_start_end
```

Comment:
Also I think the parameter can be also named just `years` - clear enough and shorter.